internal/sync/cache.go

```diff
@@ -389,13 +389,15 @@ func (c *liveStateCache) loadCacheSettings() (*cacheSettings, error) {
 	return &cacheSettings{clusterSettings, ignoreResourceUpdatesEnabled}, nil
 }
 
-// parseResourceFilter parse the given rules to generate the
+// TODO: move to shared package (util?) since we also use this for riders
```

Comment:
Since we're formatting `permittedRiders` the same way as `includedResources` we should also use the same functions and structs to evaluate them. In that case, we could move these all to a shared package.

Reply:
I guess you will need to move the `ResourceType` struct there too